### PR TITLE
added hide card glow option in settings

### PR DESCRIPTION
**Pull request by MominRaza**

### CODE REVIEW

File: lib/localization/app_en.arb
Comment (MominRaza) [inline lib/localization/app_en.arb:1]: @h4h13 I need your feedback on the wording here

**Comment by h4h13** [comment]
I have removed this card glow now it will be plain old card list 
image

**Comment by MominRaza** [comment]
You are planning to remove this card glow feature completely, So that means I can stop working on this setting option

**Comment by h4h13** [comment]
I made an option to switch between the old style to new list account style

**Comment by h4h13** [comment]
Closing this PR as I have added more blur to the card  to readable text

**Pull request closed without merging**